### DISCUSSION TITLE
Forgot Password - different response time for existing and non-existing users - possible vulnerability

### DISCUSSION THREAD

**Discussion by bedo5** [keycloak/keycloak#28963] [Q&A] [answered]
Keycloak 23.0.7.
In the dialog **Forgot Your Password?** when I enter an existing user, the response time is 500-600 ms. When I enter a non-existent one, the response time is 50-60 ms. Accordingly, it is possible to find out a valid login name.
Is there anything I can configure to make it undetectable?

**Comment by codespearhead** [comment, marked as answer]
[This answer](https://github.com/keycloak/keycloak/discussions/28172#discussioncomment-9012545) also serves as the answer to this discussion.
